### DISCUSSION TITLE
Issue with files

### DISCUSSION THREAD

**Discussion by ulisesmonge** [BearWare/TeamTalk5#1232] [Q&A]
I.
I don't know when this started, but It's something very strange. In my TeamTalk server, previowsly 5.8.3, updated to 5.9.0 beta to see if this is fixed, a user can only see files on chanels from the same username. I have 2 admins, and a anonimous account. If I as an admin upload a file, only I can see it. If an anonimous user uploads a file, only the anonimous users can see it, but I can't, undless I enter as anonimous. Someone knows what could be happening? I used a TTMediaBot to use music but I tried to log it out from the server but nothing happens, so I don't think that is the problem. Someone is experiencing this? Thank you!

**Comment by bear101** [comment]
Which client are you using? iOS, Android, Windows? Default client or Classic?
